File: Java/Limas_Segitiga.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package limas_segitiga;
import java.util.Scanner;
/**
 *@author RedLeo
 **/
public class Limas_Segitiga {
    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
       double V, L, a, l, t, T;
       final double set = 0.5, sep = 0.33;
       String nama="YUSUF RAFLI B", nim="21120118130101";
       int shift = 4, kelompok = 51;
       
       System.out.println("Saya "+nama+" dengan NIM "+nim+".\nKelompok "+kelompok+", Shift "+shift+".\n");
       
       Scanner input = new Scanner(System.in);
       System.out.println("Alas = ");
       a = input.nextDouble();
       System.out.println("Tinggi Segitiga = ");
       t = input.nextDouble();
       System.out.println("Tinggi Limas = ");
       T = input.nextDouble();
       L=set*a*t;
       V=sep*L*T;
       
       System.out.println("Luas Segitiga = 1/2 * a * t\nLuas Segitiga = 1/2 * "+a+" * "+t+" = " +L);
       System.out.println("Volume Segitiga = 1/3 * Luas alas * Tinggi Limas\nVolume Segitiga = 1/3 * "+L+" * "+T+" = "+V);
    }
    
}

File: C#/Limas_Segitiga.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Limas_Segitiga
{
    class Program
    {
        static void Main(string[] args)
        {
            float luas, alas, tinggi, tinggi_, volume;
            const float set = 0.5f, sep = 0.33f;
            string nama = "YUSUF RAFLI B", nim = "21120118130101";
            int shift = 4;//, Kelompok=51;
            char[] Kelompok = { 'L', 'I' };

            Console.Write("Nama : {0}, NIM : {1}\nKelompok : {2}{3}, Shift : {4}\n", nama, nim, Kelompok[0], Kelompok[1], shift);
            Console.Write("Masukkan alas : ");

            while (!float.TryParse(Console.ReadLine(), out alas))
            {
                Console.WriteLine("Format Masukan Salah!\nMasukan kembali nilai alas : ");
            }
            Console.Write("Masukkan tinggi segitiga : ");
            while (!float.TryParse(Console.ReadLine(), out tinggi))
            {
                Console.WriteLine("Format Masukan Salah!\nMasukan kembali nilai tinggi segitiga : ");
            }
            Console.Write("Masukkan tinggi limas : ");
            while (!float.TryParse(Console.ReadLine(), out tinggi_))
            {
                Console.WriteLine("Format Masukan Salah!\nMasukan kembali nilai tinggi limas : ");
            }
            
            luas = alas * tinggi * set;
            volume = sep * luas * tinggi_;
            Console.WriteLine("Luas Limas Segitiga = 1/3 * luas alas * tinggi limas\nLuas Limas Segitiga = 1/3 * {0} * {1} = {2}", luas, tinggi_, volume);

            Console.ReadKey();
        }
    }
}
